sys/net/network_layer/fib/fib.c

```diff
@@ -625,6 +643,13 @@ static void fib_print_adress(universal_address_container_t *entry)
     uint8_t *ret = universal_address_get_address(entry, address, &addr_size);
 
     if (ret == address) {
+#ifdef MODULE_NG_IPV6_ADDR
```

Comment:
But that's only because `ng_ipv6` pulls in `ng_ipv6_addr` as dependency. Guarding it with `MODULE_NG_IPV6_ADDR` is the right thing here because only functions from hat module are used here. With the guard below I don't exactly get why there is a guard.